Documentation/guides/messages/xa0118.md

```diff
@@ -0,0 +1,12 @@
+# Compiler Warning XA0118
+
+These warning indicate either that the Xamarin.Android project has a value in
+the `$(NuGetTargetMoniker)` MSBuild property that NuGet does not recognize or
+that the *obj\\project.assets.json* file generated by the NuGet build targets
+for the project is missing a `targets` section that matches the
+`$(NuGetTargetMoniker)`. These warnings should not arise in normal use.
```

Comment:
I would break the previous sentence up in to two at the `or`. Its currently difficult to read.